### ISSUE TITLE
Improve colour of group highlight in side nav

### ISSUE THREAD

**Issue by richard-cox** [rancher/dashboard#9799]
- Following the changes to the side-nav we now show the entire expanded side nav group as highlight
- The colour for this in light mode is a subtle grey, this looks ok
- The colour in non-light mode (dark mode, and `suse` and `csp` branding) is much less subtle

light mode
![image](https://github.com/rancher/dashboard/assets/18697775/0f0a19fb-d1ba-4199-9ae4-6b4bdc7f17fa)

Dark Mode
![image](https://github.com/rancher/dashboard/assets/18697775/f7c28849-2aea-44eb-ba2e-a10914218f42)

`suse` branding
![image](https://github.com/rancher/dashboard/assets/18697775/1892a30b-f07f-4393-82e7-a06b6e09341c)

`csp` branding
![image](https://github.com/rancher/dashboard/assets/18697775/1293c17e-dfae-45c6-94a8-ba7b43ff0f7e)

**Comment by nwmac** [comment]
@kwwii Ahead of reviewing the theme colours altogether, can we make a tactical change to this colour for dark mode for 2.11.

I'd suggest two options, for all of our current themes:

1. Grey - #333

![Image](https://github.com/user-attachments/assets/fb7fd4df-6541-498d-af24-ad96c6ae5f00)

2. Black - #000

![Image](https://github.com/user-attachments/assets/c41173a5-3888-449c-a6af-3ac57aebd0b1)

Currently we use a colour based off of the primary colour and for the Prime theme, this looks pretty awful.

**Comment by kwwii** [comment]
I completely agree; this is a problem with the dark theme. I think that the black might provide too little contrast, and I would suggest going with the grey Neil suggested as a quick fix. We will revisit this soon when we look into changing the themes fundamentally.

**Comment by nwmac** [comment]
Thanks @kwwii, that was my thought too.

**Comment by a-arias** [comment]
**TESTS**
 Rancher : v2.11-efae3660c7aa2bf92fbd1310dcc3972bcde44133-head
UI: master https://github.com/rancher/dashboard/commit/d240d2a300d0df2ae1d1cb348510862c71170eb7

Fix verified, now we have the color suggested ✅ 


<img width="1052" alt="Image" src="https://github.com/user-attachments/assets/88961346-7f3e-4026-8fe1-7b174eded26f" />
